src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotifications.java

```diff
@@ -63,11 +63,15 @@
     private static final String STOPPED_STATE = "STOPPED";
     private static final String INPROGRESS_STATE = "INPROGRESS";
 
-    private static String getRootURL(@NonNull Run<?, ?> build) {
+    private static String getRootURL() {
+        return DisplayURLProvider.get().getRoot();
+    }
+
+    private static String getRunURL(@NonNull Run<?, ?> build) {
```

Comment:
This method was confusing - it was returning job url but saying it was returning root url, and since I needed a new method that would return a root url - I had to rename this.